### PR TITLE
feat: Integration delete command added

### PR DESCRIPTION
**Pull request by ali-smartbear**
- Adding the integration:delete command.
- Deletes the given Integration from the given API
- Added delete method to API requests

### CODE REVIEW

File: src/commands/integration/delete.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    return this.executeHttp({
+      execute: () => deleteApi([owner, api, version, 'integrations', integrationId]), 
+      onResolve: this.logCommandSuccess({ apiPath: [owner, api, version].join('/') }),
+      options: {}
+    })
+  }  
+}
+
+DeleteIntegrationCommand.description = 'removes the given integration from the given API.'
```
Comment (conor-ward) [inline src/commands/integration/delete.js:24]:
```suggestion
DeleteIntegrationCommand.description = 'deletes the integration from the given API.'
```

File: src/commands/integration/delete.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+DeleteIntegrationCommand.examples = [
+  'swaggerhub integration:delete organization/api/1.0.0/503c2db6-448a-4678-a310-f465429e9704'
+]
+
+DeleteIntegrationCommand.args = [{ 
+  name: 'OWNER/API_NAME/VERSION/INTEGRATION_ID',
+  required: true,
+  description: 'Remove integration from the API'
```
Comment (conor-ward) [inline src/commands/integration/delete.js:33]:
```suggestion
  description: 'Integration to delete'
```

File: src/requests/api.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
-const { getSpec, putSpec, postSpec } = require('./spec')
+const { getSpec, putSpec, postSpec, deleteSpec } = require('./spec')
 
 const getApi = (pathParams, queryParams, accept = 'json') =>
   getSpec('apis', pathParams, queryParams, accept)
 
+const deleteApi = pathParams =>
+deleteSpec('apis', pathParams)
```
Comment (conor-ward) [inline src/requests/api.js:7]:
```suggestion
  deleteSpec('apis', pathParams)
```

**Review by conor-ward — COMMENTED** [review]
Looks good

**Comment by barryadk** [comment]
Could you update your branch and regenerate the README.md to include your recent changes?

File: test/commands/integration/delete.test.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+describe('invalid integration:delete command issues', () => {
+  test
+    .command(['integration:delete'])
+    .exit(2)
+    .it('runs integration:delete with no identifier provided')
+
+  test
+    .command(['integration:delete', 'owner/api/version'])
+    .exit(2)
+    .it('runs integration:delete with api version identifier provided')
```
Comment (barryadk) [inline test/commands/integration/delete.test.js:38]:
```suggestion
    .it('runs integration:delete without integration identifier provided')
```

File: src/template-strings/info.js [hunk 5]
```diff
@@ -23,6 +23,8 @@ const infoMsg = {
 
   IntegrationCreate: 'Created integration on API \'{{apiPath}}\'',
 
+  IntegrationDelete: 'Deleted integration from API \'{{apiPath}}\'',
```
Comment (barryadk) [inline src/template-strings/info.js:26]: Could you follow the format
`'Deleted integration \'{{integrationId}}\' on API \'{{apiPath}}\'',`

**Review by barryadk — APPROVED** [review]

**Review by conor-ward — APPROVED** [review]

**Pull request merged**